test/components/for_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Surface.Components.ForTest do
+  use ExUnit.Case, async: true
+
+  alias Surface.Components.For, warn: false
+
+  import ComponentTestHelper
+
+  describe "Without LiveView" do
```

Comment:
I think we don't need that `describe` anymore. I used to use this in some old tests for reasons I don't even remember :)